src/main/java/edu/uc/inventorymanager/dao/UserDAOStub.java

```diff
@@ -18,4 +24,14 @@ public User save(User user) throws Exception {
     public List<User> fetchAll() {
         return allUsers;
     }
+
+    @Override
+    public List<User> fetchUsers(String category) throws IOException {
```

Comment:
This implementation should be in a separate UserRetrofitDAO class that calls the UserDAO and builds and returns the JSON

Reply:
I followed the video in module 7 to the best of my understanding so I am not quite understanding what you mean

Reply:
It'll make a bit more sense once you merge in the data persistence code. Just like I made a UserSQLDAO, you'll want a UserRetrofitDAO so that we can choose to list users as JSON or we can list them as an ArrayList for the frontend